Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -33,8 +33,11 @@ public class PayPalClient {
     /// Present PayPal Paysheet and start a PayPal transaction
     /// - Parameters:
     ///   - presentingViewController: the ViewController to present PayPalPaysheet on, if not provided, the Paysheet will be presented on your top-most ViewController
-    ///   - delegate: Completion block to handle buyer's approval, cancellation, error, create order callback, shipping change callback
-    public func start(presentingViewController: UIViewController? = nil, orderID: String, delegate: PayPalDelegate?) async {
```

Comment:
@jcnoriega we want to take away the ability to pass delegate in method?
same for a simpler method with orderId, we just want one single method with a callback?

Reply:
In our other clients the delegate is set separately, this is consistent with what we have.
If we do a separate method for orderId, we should do it for baToken and vaultIDs, which in essence are just other types of ids. And its more consistent with the real NXO interface